validator/validator-main.protoascii

```diff
@@ -4460,7 +4460,6 @@ attr_lists: {
   attrs: { name: "placeholder" value: "" }
   # We allow the empty fallback attribute on any element
   attrs: { name: "fallback" value: "" }
-  attrs: { name: "lightbox" }
```

Comment:
This change was already submitted on G3 by @Gregable. Does it automatically get sync-ed here / should I rebase / wait for it?

Reply:
You'll need to pull it into your branch. I believe that would just be a fetch/merge to that branch.

Reply:
I pulled this to GH yesterday, so just sync?